packages/block-library/src/gallery/gallery.js

```diff
@@ -23,26 +25,27 @@ export const Gallery = ( props ) => {
 		className,
 		isSelected,
 		setAttributes,
-		selectedImage,
 		mediaPlaceholder,
-		onMoveBackward,
-		onMoveForward,
-		onRemoveImage,
-		onSelectImage,
-		onDeselectImage,
-		onSetImageAttributes,
 		onFocusGalleryCaption,
 		insertBlocksAfter,
+		images,
 	} = props;
 
 	const {
 		align,
-		columns = defaultColumnsNumber( attributes ),
+		columns = defaultColumnsNumber( images ),
 		caption,
 		imageCrop,
-		images,
 	} = attributes;
-
+	const innerBlocksProps = useInnerBlocksProps(
+		{
+			className: classnames( 'wp-block-gallery__container' ),
```

Comment:
Nit; `classnames()` not needed for just one classname?

Reply:
fixed